core/builder/src/test/java/io/quarkus/builder/BasicTests.java

```diff
@@ -262,7 +262,6 @@ public void execute(final BuildContext context) {
             fail("Expected exception");
         } catch (ChainBuildException expected) {
             // ok
-            assertTrue(expected.getMessage().contains("overridable"));
```

Comment:
I think this test class can be removed entirely?

Reply:
Well we want to be sure that the old API still works because it's still used by hundreds of classes.